Client/mods/deathmatch/logic/CClientVectorGraphic.h

```diff
@@ -36,7 +36,7 @@ class CClientVectorGraphic final : public CClientTexture
     lunasvg::Document* GetSVGDocument() const { return m_pSVGDocument.get(); }
     CXMLNode*          GetXMLDocument() const { return m_pXMLDocument; }
 
-    CClientVectorGraphicDisplay* GetDisplay() const { return m_pVectorGraphicDisplay.get(); }
```

Comment:
And you can return shared_ptr here, so CClientVectorGraphicDisplay will be destroyed when all references are invalidated.

Reply:
I didn't return shared_ptr here, because the pointer returned is always discarded on the same frame (by design/usage). Plus, there would still be a reference in the display manager at this time, so it wouldn't make a difference.